.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "python.pythonPath": "C:\\Program Files\\Python38\\python.exe"
```

Comment:
Can you remove that from your pull request? 
Ideally just add a ".vscode" in .gitignore if that is important for you.